Jenkinsfile

```diff
@@ -82,6 +82,9 @@ pipeline {
                     script {
                         env.TAG = sh(script: "grep Version DESCRIPTION | head -n1 | cut -d':' -f2", returnStdout: true).trim()
                     }
+                    
+                    // this solves git's "dubious ownership" complaint, but we have no idea why and how only this repo is suddenly affected
+                    sh "git config --global --add safe.directory /home/jenkins/agent/workspace/nis_molgenis-r-datashield_master"
```

Comment:
does this safe.directory not have to be: /home/jenkins/agent/workspace/molgenis_molgenis-r-datashield_master?

Reply:
nevermind that, do as the jenkins has ordered